pkg/odo/cli/component/exec.go

```diff
@@ -61,7 +61,8 @@ Please provide a command to execute, odo exec -- <command to be execute>`)
 		return fmt.Errorf("no parameter is expected for the command")
 	}
 
-	eo.devfilePath = filepath.Join(eo.componentContext, eo.devfilePath)
+	eo.devfilePath = filepath.Join(eo.componentContext, devFile)
```

Comment:
I think we should be using `DevfilePath` from create.go

Reply:
yeah i thought about it, but in create.go
`var DevfilePath = filepath.Join(LocalDirectoryDefaultLocation, devFile)`

and its only changing in Complete():
```
if co.componentContext != "" {
			DevfilePath = filepath.Join(co.componentContext, devFile)
```

which is correct since odo commands can run with any context during their execution.

Reply:
OK